srcpkgs/libco/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'libco'
+pkgname=libco
+version=1.0.2
+revision=1
+build_style=gnu-makefile
+short_desc="Cooperative multithreading library written in C89"
+maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+license="Public Domain"
+homepage="https://github.com/freeekanayaka/libco"
+distfiles="https://github.com/freeekanayaka/libco/archive/v${version}.tar.gz"
+checksum=8876aa8e61f07b37adb3ad422b369ecf9bf6fad20ce691428c35719ba5f16615
+shlib_provides="libco.so"
+
+do_install() {
+	vinstall libco.so 0664 /usr/lib/ libco.so.${version}
```

Comment:
This looks an awful lot like you're taking sonames of variable versions and linking them to a fixed .1 version.  This is almost certainly wrong.